### PR TITLE
make:docs: add symlinks

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -325,10 +325,16 @@ signin:
...
-run: init
+run: docs init
 	@echo 'run'
 	$(DOCKER_COMPOSE) $(VERBOSE) $(NOCACHE) up --remove-orphans &
 	@echo ''
+.PHONY: docs
+docs:
+	install -v README.md docs/docs/index.md
+	sed 's/\/images/.\/images/' README.md > docs/docs/index.md
+	install -v images/* docs/docs/assets/images
```
Comment (RandyMcMillan) [inline GNUmakefile:336]: using sed to fix image link

File: README.md [hunk 2]
```diff
@@ -146,5 +146,5 @@ bos --version
 - John Doe
 - @rafgxyz
 - [@asherp](https://github.com/asherp)
-- [@RandyMcMillian](https://github.com/randymcmillan)
-- @nitesh_btc
+- [@RandyMcMillan](https://github.com/randymcmillan)
```
Comment (RandyMcMillan) [inline README.md:149]: fix spelling

File: docs/docs/index.md [hunk 3]
```diff
@@ -1,4 +1,150 @@
----
-template: overrides/home.html
-title: Material for MkDocs
----
+![create lnd wallet image](./images/PlebnetPlayground.png)
```
Comment (RandyMcMillan) [inline docs/docs/index.md:1]: replaced by sed in makefile

File: README.md [hunk 4]
```diff
@@ -146,5 +146,5 @@ bos --version
...
+- @nitesh_btc
```
Comment (RandyMcMillan) [inline README.md:150]: :)

**Comment by rsafier** [comment]
this ready to merge or you have some more?

**Comment by RandyMcMillan** [comment]
#13 is better.

**Pull request closed without merging**